CODE_REVIEW.md

```diff
@@ -0,0 +1,46 @@
+
+# Code Review
+
+## Problems 
+1. Potential memory leak from observable in `ngOnInit()` in `libs/books/feature/src/lib/book-search/book-search.component.ts`. The observable should be unsubscribed from using `ngOnDestroy()` **--Fixed**
+
+2. Files with different functions are grouped together in `libs/books/data-access/src/lib/+state`. Actions, reducer and selector files should be grouped according to their function and their reference undated everywhere
+
+3. No type tags added in `nx.json` and defined in `tslint.json`. Tags should be added to control folder access between apps in a mono repo
+
+4. This file `test-setup.ts` exists in both the `apps` and `libs` folders (`apps/okreads/browser/src/` and `libs/books/feature/src/`). It should be removed from `apps` and    should only exist in `libs` since it is shared.
+
+5. Bad naming conviention of `fb: FormBuilder` in `libs/books/feature/src/lib/book-search/book-search.component.ts`. Avoid using single and double character name declaration. Give specific names to variables and methods **--Fixed**
+
+6. Access modifiers like `private` are not used in class variables to encourage encapsulation. **--Fixed**
```

Comment:
Where have you fixed this? I can't see any changes.

Reply:
I actually double checked this and discovered it was not necessary to add a private modifier to any of the variables. But I made the Subject (for the "takeUntil()" operator) I mentioned earlier a private variable in book-search.component.ts